src/Composition/ForceElements/SpringDamperPtP.jl

```diff
@@ -37,20 +37,20 @@ mutable struct SpringDamperPtP <: Modia3D.AbstractForceElement
 
     function SpringDamperPtP(; obj1::Object3D,
                                obj2::Object3D,
-                               nominalLength::Float64 = 0.0,
-                               nominalForce::Float64 = 0.0,
-                               springForceLaw::Union{Float64, Function} = 0.0,
-                               damperForceLaw::Union{Float64, Function} = 0.0 )
+                               nominalLength::Real = 0.0,
+                               nominalForce::Real = 0.0,
+                               springForceLaw::Union{Real, Function} = 0.0,
+                               damperForceLaw::Union{Real, Function} = 0.0 )
```

Comment:
Why do you use `Real` here, isn't `Float64` sufficient?

Reply:
This is like in Revolute.jl and Prismatic.jl now: The user can pass any real number (Int, Float32,...). Internally we always use Float64.